### PR TITLE
WIP: Avatars, Badges and Buttons docs

### PR DESCRIPTION
**Pull request by shivampandey0**
- fix: Icon positioning fix inside `badge-wrapper`
- doc: Avatar components added in docs with examples (It was supposed to go in previous PR but due to late PR review it got pushed to this)
- doc: Badge components added in docs with examples
- doc: Button components added in docs with examples

## [Preview link](https://marvel-ui.vercel.app/index.html)

### CODE REVIEW

**Review by saadahmadhere — APPROVED** [review]
Looks good to me.

File: css/helpers/flexbox.css [hunk 1]
```diff
@@ -39,19 +39,19 @@
 
 /* flex gaps */
 .gap-05 {
-  gap: var(--size-000);
+  gap: var(--scale-05);
 }
```
Comment (Hamdrive) [inline css/helpers/flexbox.css:43]: for the classname on line 41, using `05` would confuse someone as `5`
Suggest to use some variable instead , like `xs` or `sm`

File: docs/avatars.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
```
Comment (Hamdrive) [inline docs/avatars.html:1]: Why is avatar HTML in this branch? Name of branch is for Badge and Button
Suggest to create another branch and append Avatar changes there, as only max 2 components can be on one branch

**Review by Hamdrive — CHANGES_REQUESTED** [review]
Overall code looks clean!

Suggest to move the avatar HTML into its own branch, as this doesnt comply with rule of max 2 components in one branch

File: docs/avatars.html (hunk 2 above)
Comment (shivampandey0) [reply docs/avatars.html:1]: hey, I made all the components earlier in their individual branch. Right now I'm doing docs only for the components that were already made in their individual branches. 
But doing PR only 2 components at a time from the `docs` branch.

File: css/helpers/flexbox.css (hunk 1 above)
Comment (shivampandey0) [reply css/helpers/flexbox.css:43]: hey, it's a scale table defined in variables and I'll make a naming convention page in docs to better understand. 
As of now, you can refer to` reset.css` to check on this.

Comment (Hamdrive) [reply css/helpers/flexbox.css:43]: Understood, we discussed and you mentioned that you will refactor it lateron, 
we can close this discussion 👍

File: docs/avatars.html (hunk 2 above)
Comment (Hamdrive) [reply docs/avatars.html:1]: Discussed and proposed suggesting to change PR title and description. 
Approved and can close this conversation 👍

**Review by Hamdrive — COMMENTED** [review]
Discussed on call, changes to naming convention will be mentioned in docs and can be refactored lateron.
Title and description of the PR will be changed as discussed to reflect all major files being refactored.

Approved these changes

**Comment by shivampandey0** [comment]
Discussed with Reviewer, explained the rationale behind the code, and refactored title and description proposed by Reviewer.

**Pull request merged**